supernode/daemon/mgr/cdn/manager.go

```diff
@@ -103,18 +104,16 @@ func (cm *Manager) TriggerCDN(ctx context.Context, task *types.TaskInfo) (*types
 
 	return &types.TaskInfo{
 		CdnStatus:  types.TaskInfoCdnStatusSUCCESS,
-		FileLength: realFileLength,
+		FileLength: downloadMetadata.realFileLength,
 		RealMd5:    realMD5,
 	}, nil
 }
 
 // GetHTTPPath returns the http download path of taskID.
+// The returned path joined the DownloadRaw.Bucket and DownloadRaw.Key.
 func (cm *Manager) GetHTTPPath(ctx context.Context, taskID string) (string, error) {
-	info, err := cm.cacheStore.Stat(ctx, getDownloadRawFunc(taskID))
-	if err != nil {
-		return "", err
-	}
-	return info.Path, nil
+	raw := getDownloadRawFunc(taskID)
+	return path.Join("/", raw.Bucket, raw.Key), nil
```

Comment:
use `filepath.Separator` instead `/`

Reply:
In fact, it's a http path not file path.